ui/components/app/mmi/interactive-replacement-token-notification/interactive-replacement-token-notification.component.js

```diff
@@ -0,0 +1,117 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { sha256 } from '../../../../../shared/modules/hash.utils';
+
+export default class InteractiveReplacementTokenNotification extends PureComponent {
```

Comment:
Please refactor all new components to be functional components, adding more class based components is increasing our technical debt.